e2e/testdata/fn-render/kptfile-unknown-fields/.expected/config.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# One of the functions in the pipeline fails resulting in
+# non-zero exit code and no changes in the resources.
+exitCode: 1
+stdErr: |
+  yaml: unmarshal errors:
+    line 13: field targetResources not found in type v1.Function
```

Comment:
link to the function schema would be helpful (https://kpt.dev/reference/schema/kptfile/).
nit: end with newline
We can improve the message further when we release target selectors feature.

Reply:
Fixed the newline.
Yes, we can improve the message as we go along. Also realized we need to improve the schema page for `Kptfile`. It is not useful in the current form. Only helpful in grabbing schema.